country-a-service/nsp-client/src/main/java/dk/nsp/epps/client/NspClient.java

```diff
@@ -26,7 +26,7 @@ private NspClient() {
      * Send a SOAP request to an NSP service.
      */
     public static Reply request(URI uri, Element soapBody, String soapAction, Identity caller, Element... extraHeaders) throws Exception {
-        try (var client = NspSoapClient.builder().withLogger(log::info).build();
+        try (var client = NspSoapClient.builder().withLogger(log::debug).build();
```

Comment:
This should probably be defined somewhere for all the methods, instead of inline? Or is it just some debugging that's still present?

Reply:
I'd argue that *this* is where we do it for all the methods.  This is the only instance of `NspSoapClient` that were making in the code.

The effect of this change is that we do not see all the logging from NspSoapClient unless we enable debug logging.

The whole setup is a bit icky, and it will need to change once the auth story is crystalising – this client nsp client is from the `dk.nsp.test.idp` package which is only for contacting the test environments, not the production environment.  We are going to need something that can work with both, and at that time we should be more careful about not logging PII.